coderd/database/migrations/000256_workspace_agent_script_timings.up.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE workspace_agent_scripts ADD COLUMN id           uuid UNIQUE NOT NULL DEFAULT gen_random_uuid();
+ALTER TABLE workspace_agent_scripts ADD COLUMN display_name text        NOT NULL DEFAULT '';
+
+CREATE TYPE workspace_agent_script_timing_stage AS ENUM (
+    'start',
+    'stop',
+    'cron'
+    );
+
+CREATE TABLE workspace_agent_script_timings
+(
+    script_id     uuid                                NOT NULL REFERENCES workspace_agent_scripts (id) ON DELETE CASCADE,
+    display_name  text                                NOT NULL,
+    started_at    timestamp with time zone            NOT NULL,
+    ended_at      timestamp with time zone            NOT NULL,
+    exit_code     int                                 NOT NULL,
+    stage         workspace_agent_script_timing_stage NOT NULL,
+    timed_out     bool                                NOT NULL
+);
```

Comment:
Is GitHub rendering this incorrectly or are these actually in the DOWN migration? 🤔

Reply:
This was resolved after I submitted my PR, I believe GitHub went to happy hour a bit early this Friday.